src/insights_client/tests/test_metrics.py

```diff
@@ -37,20 +38,16 @@ def factory(**config):
         config = u"""[server]
```

Comment:
There are no tests that would verify a case when this whole _server_ section would be missing.